source/grid/tria_accessor.cc

```diff
@@ -2750,15 +2727,15 @@ CellAccessor<dim, spacedim>::periodic_neighbor_of_coarser_periodic_neighbor(
   for (unsigned int i_subface = 0; i_subface < parent_face_it->n_children();
        ++i_subface)
     if (parent_face_it->child_index(i_subface) == my_face_index)
-      return (std::pair<unsigned int, unsigned int>(face_num_of_nb, i_subface));
+      return (std::make_pair(face_num_of_nb, i_subface));
```

Comment:
Why are there additional round brackets? Also I am wondering if we could simply use curly brackets like this:
```suggestion
      return {face_num_of_nb, i_subface};
```

Reply:
I agree that the braces should go, but I am not sure that brackets are easier to read than make_pair. The whole file is full of make_pair calls, that we would need to change.